libraries/src/Extension/ExtensionHelper.php

```diff
@@ -311,6 +311,7 @@ class ExtensionHelper
         ['plugin', 'checkfiles', 'task', 0],
         ['plugin', 'requests', 'task', 0],
         ['plugin', 'sitestatus', 'task', 0],
+        ['plugin', 'globalcheckin', 'task', 0],      
```

Comment:
```suggestion
        ['plugin', 'checkfiles', 'task', 0],
        ['plugin', 'globalcheckin', 'task', 0],
        ['plugin', 'requests', 'task', 0],
        ['plugin', 'sitestatus', 'task', 0],
```
Alpha-ordering + code style fix (no white space at end of line)